index.html

```diff
@@ -5,9 +5,40 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Portfolio</title>
 </head>
 <body>
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <nav id="Logo-Placeholder">
+             <a href="#" id="My-Logo"><img src="img/My_logo.png" alt="Home page"></a>
+         </nav>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
```

Comment:
`<nav id="Normal-Button-Tertiary-Enabled">         
            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
         </nav>`

- [ ] Just as the previous reviewer requested, wrap the hamburger menu icon in a nav tag, currently you wrapped the logo in the nav tag and not the menu icon, please change this 👍🏿